<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SwitchPushButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="InformationPage">
     <widget class="QLabel" name="label_text_dorm_find">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>住宿信息查询</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_stuid"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_find_by_stuid">
         <property name="text">
          <string>按学号查找学生</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="listView_dorm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>71</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_text_student_info">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>住宿人员信息:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dorm_number">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_to_dorm">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>跳转到宿舍</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_students">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>511</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_text_dorm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>宿舍号:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ManagePage">
     <widget class="QTabWidget" name="MangeTabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>691</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="OutsiderComeTab">
       <attribute name="title">
        <string>外访登记</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ThingsMangeTab">
       <attribute name="title">
        <string>公用物品管理</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="RepairTab">
       <attribute name="title">
        <string>维修申请登记</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="GoAndLeaveTab">
       <attribute name="title">
        <string>留校/离校登记</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="RuleBreakRecordTab">
       <attribute name="title">
        <string>违纪登记</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLabel" name="label_text_function_manage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>功能管理</string>
      </property>
     </widget>
     <zorder>label_text_function_manage</zorder>
     <zorder>MangeTabWidget</zorder>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_todo_things">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>296</width>
      <height>251</height>
     </rect>
    </property>
    <property name="dateEditEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_todolist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>备忘录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
